docs/posix-secondaries.adoc

```diff
@@ -84,3 +88,22 @@ $AKT_SECNDR -c $AKT_PROJ_HOME/config/posix-secondary.toml
 ....
 * Add --loglevel 0 to the aforementioned launch commands if you would like to see more logs
 * To re-register your emulated multi-ECU device (or start playing it from scratch) remove ``storage`` directory from `PRIMARY_HOME_DIR`
+
+=== Multiple secondaries
+
+In order to emulate a device containing one primary ECU along with more than one secondary ECUs the following should be done.
+
+* Run the secondary executable ``<build-dir>/src/aktualizr_secondary/aktualizr-secondary desired number of times each from different directories.
```

Comment:
I think you're missing the closing backtick for the path.

Reply:
Yes, just updated.